l10n_es_account_bank_statement_import_n43/models/account_bank_statement_import_n43.py

```diff
@@ -245,13 +245,41 @@ def _get_partner_from_santander(self, conceptos):
                     partners = partner_obj.search([('name', 'ilike', name)])
         return partners and partners[0].id or False
 
+    def _get_partner_from_bankia(self, conceptos):
+        partner_obj = self.env['res.partner']
+        partners = []
+        # Try to match from partner name
+        if conceptos.get('01'):
+            vat = conceptos['01'][0][:2] + conceptos['01'][0][7:]
+            if vat:
+                partners = partner_obj.search([('vat', '=', vat)])
```

Comment:
Ah, genial entonces. Ya por curiosidad, qué viene entre medias del ES y el NIF que tenéis que quitarlo?